github-actions/verify-pr-title/src/index.ts

```diff
@@ -19,30 +21,43 @@ function run(): void {
 		// Examine the title for the expected patterns, which vary depending on what the target branch is.
 		const title = pullRequest.title as string;
 		const base = pullRequest.base as {ref: string};
-		const targetBranch = base.ref;
-		if (targetBranch === "release" || targetBranch === "documentation") {
-			// Release branches have their own less stringent format.
-			if (verifyReleaseBranchPrTitle(title)) {
-				console.log(`PR title '${title}' accepted for release branch.`);
+		const baseBranch = base.ref;
+		const head = pullRequest.head as {ref: string};
+		const headBranch = head.ref;
+		if (headBranch === "main" && baseBranch === "dev") {
```

Comment:
Do we care about dev-3?

Reply:
No branch based off `dev` should ever be merged into `dev-3`, so there's no need to account for it here.
We could have a companion PR to this one that makes the equivalent changes in `dev-3` though, if we want.